content/stylesheets/pages/_awards.scss

```
@@ -0,0 +1,98 @@
+.awards-page {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+
+  .sub-heading {
+    font-size: 1.2rem;
+    color: var(--color--secondary);
+    text-align: center;
```

```suggestion
    text-align: start;
```

I am thinking maybe the sub-heading better be left-aligned?